### DISCUSSION TITLE
How to install ver1.2.0 for all users

### DISCUSSION THREAD

**Discussion by Atechguy95** [mu-editor/mu#2413] [Q&A]
Hello!

I have downloaded Mu on a local account but the app is not showing up for domain users. The install package does not give the option to install for all users. I tried to manually move its file location to programs instead of users but it still isn't showing up for my domain users. There was a support article from 2021 but I didn't find any helpful information on it. Probably operator error but any help is appreciated!

**Comment by dsssssssss9** [comment]
Hi 

I had this a long time ago with a different program , turns out it was because the program needed to be 1st installed under an admin account .

If you install using a standard user account the program would only be available to that user .

Hope this helps

**Reply by Atechguy95** [reply]
I installed the application while logged into our local admin account... It just isn't showing up for our domain accounts.

**Reply by PrescotAdmin** [reply]
use allusers="1" this will install it to program files(x86)

Example: msiexec /i "MuEditor-win64-1.2.0.msi" ALLUSERS="1" /qn /norestart


UPDATE - Installs it no problem but crashes on running have a bug report submitted will update my reply once I know anything.
